.travis.yml

```diff
@@ -28,6 +28,18 @@ env:
   global:
     - VERSION="13.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
 
+  matrix:
+    - LINT_CHECK="1"
+    - TESTS="1" ODOO_REPO="OCA/OCB"
+      EXCLUDE="mail_restrict_follower_selection,mail_autosubscribe"
+    - TESTS="1" ODOO_REPO="OCA/OCB" INCLUDE="mail_restrict_follower_selection"
+    - TESTS="1" ODOO_REPO="OCA/OCB" INCLUDE="mail_autosubscribe"
+    - TESTS="1" ODOO_REPO="odoo/odoo"
+      EXCLUDE="mail_restrict_follower_selection,mail_autosubscribe" MAKEPOT="1"
+    - TESTS="1" ODOO_REPO="odoo/odoo" INCLUDE="mail_restrict_follower_selection"
+      MAKEPOT="1"
+    - TESTS="1" ODOO_REPO="odoo/odoo" INCLUDE="mail_autosubscribe" MAKEPOT="1"
+
```

Comment:
```suggestion
```

I think instead of doing this in `matrix` (as in the provided 12.0 example) it has to be done in `jobs`, because travis is configured differently in 12.0.

Try to do like this instead (replacing the current `jobs` config with this):


```yml
jobs:
  include:
    # Test separately: mail_restrict_follower_selection
    - stage: test
      env:
        - TESTS=1 ODOO_REPO="odoo/odoo" MAKEPOT=1 INCLUDE="mail_restrict_follower_selection"
    - stage: test
      env:
        - TESTS=1 ODOO_REPO="OCA/OCB" INCLUDE="mail_restrict_follower_selection"
    # Test all other addons together
    - stage: test
      env:
        - TESTS=1 ODOO_REPO="odoo/odoo" MAKEPOT=1 EXCLUDE="mail_restrict_follower_selection"
    - stage: test
      env:
        - TESTS=1 ODOO_REPO="OCA/OCB" EXCLUDE="mail_restrict_follower_selection"
```

Based on https://github.com/OCA/oca-addons-repo-template/blob/master/.travis.yml.jinja


Note that on this example I'm only excluding `mail_restrict_follower_selection` from the general tests, because it's the only one doing blocking things IMO (ie: preventing subscribing partners). `mail_autosubscribe` is not doing that and should play nicely with others.

Reply:
Thanks @ivantodorovich i thinks the same as you about to  only exclude `mail_restrict_follower_selection`.